### ISSUE TITLE
Missing DCIM

### ISSUE THREAD

**Issue by gedOHub** [kontron/python-ipmi#50]
Hello,

I was writing script which would pooling power usage from servers over IPMI and noticed that there is no DCIM section implemented in this module.

**Comment by hthiery** [comment]
You're right ... there is currently no DCIM support present. You're welcome to do this implementation. It should not be a big thing to do the support of the DCIM commands. For basic command support there are ~10 commands that has to be added.

For full native support a RMCP+ interface and SOL are also missing.

**Comment by hthiery** [comment]
just added basic support for dcim ... try it out, add your requested features and give some feadback
